client/src/utils/KBar.tsx

```diff
@@ -12,7 +12,11 @@ import { fetchWallets } from '../slices/wallets/walletsThunks'
 
 import { RenderResults } from './RenderResults'
 
-export const KBar: React.FC = ({ children }) => {
+type Props = {
+  children: JSX.Element
+}
+
+export const KBar = ({ children }: Props) => {
```

Comment:
```suggestion
export const KBar: React.FunctionComponent<PropsWithChildren> = ({ children }) => {
```

This would work right?